app/retail/emails.py

```diff
@@ -930,45 +930,46 @@ def render_new_bounty_roundup(to_email):
 </p>
 '''
     highlights = [{
-        'who': 'sulliwane',  
+        'who': 'mul1sh',  
         'who_link': True,
-        'what': 'Soccer Scores DOracle',
-        'link': 'https://gitcoin.co/issue/iExecBlockchainComputing/iexec-apps/11/3196',
+        'what': 'Deploying Full Stack DApp to AWS',
+        'link': 'https://gitcoin.co/issue/kauri-io/Content/51/3186',
         'link_copy': 'View more',
     }, {
-        'who': 'MajorTomSec',
+        'who': 'igetgames',
         'who_link': True,
-        'what': 'Copy updates for consistency',
-        'link': 'https://gitcoin.co/issue/status-im/status-react/8846/3391',
+        'what': 'Eth_gasPrice Format Example Has Leading Zeros',
+        'link': 'https://gitcoin.co/issue/ethresearch/eth-wiki/16/3439',
         'link_copy': 'View more',
     }, {
-        'who': 'batatsar',
+        'who': 'RobertMCForster',
         'who_link': True,
-        'what': 'eth_getunclebyblockhashandindex typo',
-        'link': 'https://gitcoin.co/issue/ethresearch/eth-wiki/7/3361',
+        'what': 'Linkdrop x Gnosis Safe Smart Contract Bug-bounty',
+        'link': 'https://gitcoin.co/issue/LinkdropHQ/linkdrop-safe-module/5/3309',
         'link_copy': 'View more',
     }, ]
 
     sponsor = {
-        'name': 'Blockmason',
-        'title': 'Build and deploy your whole laundry list of microservices in a single afternoon.',
-        'image_url': 'https://blockmason.link/wp-content/uploads/2018/11/logo.svg',
-        'link': 'http://bit.ly/2L5IA2n',
-        'cta': 'Get Started Now',
+        'name': 'Solana',
+        'title': 'Build on Solana: Join the Private Beta',
+        'image_url': '',
+        'link': 'http://bit.ly/solana-beta',
+        'cta': 'Join the Private Beta',
         'body': [
-            'Link creates web-based APIs from your code, so you can deploy microservices instantly.'
+            'Solana is a Lightning-fast distributed ledger technology for mission-critical decentralized apps.',
+            'We are currently taking applications for an incredibly limited beta program for early adopters of Solana to launch their projects alongside our mainnet later in 2019.'
         ]
     }
 
     bounties_spec = [{
-        'url': 'https://github.com/MetaMask/metamask-extension/issues/6699',
-        'primer': 'Malicious Dependency Update Bug Bounty',
-    }, {
-        'url': 'https://github.com/Synthetixio/synthetix/issues/196',
+        'url': 'https://gitcoin.co/issue/Synthetixio/synthetix/196/3430'
```

Comment:
Remember these have to be github links, not gitcoin! also missing comma here, note that travis failed on this PR because of this @ryan-shea